### PR TITLE
[ci_updates] pylint + Python3 compatibility - melodic

### PR DESCRIPTION
**Pull request by fmessmer**
same as #227

### CODE REVIEW

**Comment by fmessmer** [comment]
TODO: sync with #227

File: cob_hardware_emulation/scripts/emulation_follow_joint_trajectory.py [hunk 1]
```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/env python
 
+from past.utils import old_div
```
Comment (LoyVanBeek) [inline cob_hardware_emulation/scripts/emulation_follow_joint_trajectory.py]: Maybe this needs to be installed/added to deps or just undo the change related to old_dev below as well.

File: cob_twist_controller/scripts/test/test_careobot_st_jla_ca_selfcollision_arm_left_direct_head.py [hunk 2]
```diff
@@ -141,17 +142,15 @@ def init_pos():
             # save data
             for data_kraken in data_krakens:
                 data_kraken.writeAllData()
-        except rospy.ROSInterruptException as e:
```
Comment (LoyVanBeek) [inline cob_twist_controller/scripts/test/test_careobot_st_jla_ca_selfcollision_arm_left_direct_head.py:144]: Why is this bit removed?

File: cob_twist_controller/scripts/test/test_careobot_st_jla_ca_sphere.py [hunk 3]
```diff
@@ -157,23 +159,21 @@ def init_pos():
...
+            # pid.send_signal(signal.SIGINT)
             pid.kill()
-            pid.send_signal(subprocess.signal.SIGINT)
+            pid.send_signal(signal.SIGINT)
         except Exception as e:
             rospy.logerr('Failed to stop rosbag play due to exception: ' + str(e))
         try:
             traj_marker_pid.kill()
-            traj_marker_pid.send_signal(subprocess.signal.SIGINT)
+            traj_marker_pid.send_signal(subprocess32.signal.SIGINT)
```
Comment (LoyVanBeek) [inline cob_twist_controller/scripts/test/test_careobot_st_jla_ca_sphere.py]: `subprocess32` ? Where does that come from?

**Review by LoyVanBeek — CHANGES_REQUESTED** [review]

File: cob_twist_controller/scripts/test/test_careobot_st_jla_ca_sphere.py (hunk 3 above)
Comment (fmessmer) [reply cob_twist_controller/scripts/test/test_careobot_st_jla_ca_sphere.py]: was a test...I reverted it already again - but only in #227...now both PRs are in sync again

File: cob_twist_controller/scripts/test/test_careobot_st_jla_ca_selfcollision_arm_left_direct_head.py (hunk 2 above)
Comment (fmessmer) [reply cob_twist_controller/scripts/test/test_careobot_st_jla_ca_selfcollision_arm_left_direct_head.py:144]: ROSInterruptException is a child of KeyboardInterrupt...there was an error message about bad except clause order...

Comment (LoyVanBeek) [reply cob_twist_controller/scripts/test/test_careobot_st_jla_ca_selfcollision_arm_left_direct_head.py:144]: OK

File: cob_hardware_emulation/scripts/emulation_follow_joint_trajectory.py (hunk 1 above)
Comment (fmessmer) [reply cob_hardware_emulation/scripts/emulation_follow_joint_trajectory.py]: undone

**Comment by fmessmer** [comment]
- `melodic` job:
   - `pylint`: `E0401(import-error)`
   - `pylint3`: `E0401(import-error)`, `E0611(no-name-in-module)`, `E1101(no-member)`

**Review by LoyVanBeek — DISMISSED** [review]
Looks good to me
